src/main/java/net/fabricmc/stitch/util/SnowmanClassVisitor.java

```diff
@@ -45,7 +47,7 @@ public void visitLocalVariable(
 				final int index) {
 			String newName = name;
 			if (name != null && name.startsWith("\u2603")) {
-				newName = "lvt" + index;
+				newName = "local_" + localCounter++;
```

Comment:
I hope you can use camel case and preferably, name in the fashion enigma names unmapped parameters.